core/src/main/groovy/com/novoda/gradle/release/ReleasePlugin.groovy

```diff
@@ -43,8 +42,12 @@ class ReleasePlugin implements Plugin<Project> {
         }
     }
 
-    private String getString(Project project, String propertyName, String defaultValue) {
-        project.hasProperty(propertyName) ? project.getProperty(propertyName) : defaultValue
+    private String getProjectProperty(Project project, String propertyName, String defaultValue) {
+        if (project.hasProperty(propertyName) && project.getProperty(propertyName) != 'unspecified') {
+            return project.getProperty(propertyName)
+        } else {
```

Comment:
it depends, if both cases have equal weighting i.e. both just as likely to happen then `if else` is definitely valid, however if the first case is a *quick return* or *guard clause* meaning they check this case first and everything else is treated as the norm then yes you can drop the else, but you should always consider the weighting for the potential of this being an acceptable `if else`